internal/pkg/vm/internal/vmcontext/syscalls.go

```diff
@@ -18,6 +18,13 @@ var _ specsruntime.Syscalls = (*syscalls)(nil)
 
 // VerifySignature implements Syscalls.
 func (sys syscalls) VerifySignature(signature specscrypto.Signature, signer address.Address, plaintext []byte) error {
+	// Dragons: this lets all id addresses off the hook -- we need to remove this
+	// once market actor code actually checks proposal signature.  Depending on how
+	// that works we may want to do id address to pubkey address lookup here or we
+	// might defer that to VM
```

Comment:
Intention is to do the lookup here, I believe (where we can cache it). This would make the call non-pure function though. I'll confirm with lotus.

ID addresses are the *only* addresses I expect to actually see here.